documentation/book/cluster-operator.adoc

```diff
@@ -851,6 +866,16 @@ include::examples/install/cluster-operator/02-role.yaml[]
 
 When the Role cannot be created, the predefined ClusterRole named `edit` can be used instead.
 
+If the rack awareness feature is used, the started Kafka init container needs to operate using a ClusterRole that gives it access to the necessary resources.
+
+[source,yaml,options="nowrap"]
+.Example Role for the Kafka init container
+----
+include::examples/install/cluster-operator/05-role-kafka.yaml[]
+----
+
+The assigned privileges allow the Kafka init container to get Nodes information.
```

Comment:
Nodes information -> information from the nodes

Reply:
@kyguy the capital "N" should refer to the Node resource that we have in Kubernetes/OpenShift (like StatefulSets and so on), why "Nodes information" doesn't sound well?

Reply:
Ah I understand now ( I hadn't realized it was referring to the Kubernetes resource Nodes ), I thought it might have been a typo!